src/main/java/us/icarenow/web/model/entity/Role.java

```diff
@@ -0,0 +1,7 @@
+package us.icarenow.web.model.entity;
+
+public enum Role {
```

Comment:
Oare asta nu poate sa fie clasa normala?
am doua motive:
motivu real: nu am inteles niciodata care e faza cu enum
motivu oficial: nu pusca cu baza de date

Reply:
True, I remember that we did that (having Roles - enum & Role - entity), @radumbaciu Are you sure you pushed all the changes?

Reply:
not all, just the last working & tested version. Was planning on doing the other changes in a future PR

Reply:
Then please wait for Adi's changes. Review needed there